t_syntax.py

```
@@ -0,0 +1,314 @@
+
+                   SSUUMMMMAARRYY OOFF LLEESSSS CCOOMMMMAANNDDSS
```

Ruff (0.12.2)

SyntaxError: Unexpected indentation

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: Got unexpected token 

---

SyntaxError: unindent does not match any outer indentation level

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Compound statements are not allowed on the same line as simple statements

---

SyntaxError: Starred expression cannot be used here

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Got unexpected token 

---

SyntaxError: Expected an identifier

---

SyntaxError: Expected ',', found indent

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Got unexpected token 

---

SyntaxError: Expected an identifier

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found ';'

---

Multiple statements on one line (semicolon)

(E702)